### ISSUE TITLE
Instance: There is a second, focusable control in between the `.chat-thread-toggle` disclosure controls and their toggled content | WCAG 2.2 Violation - There are instances of illogical focus order present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#42]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.4.3 Focus Order](https://www.w3.org/TR/WCAG22/#focus-order)

### Screenshot

![Screenshot showing a \`.chat-thread-toggle\` (denoted via a '1'), immediately followed by another control in the source order (denoted via a '2').](https://github.com/user-attachments/assets/fc56e7db-9d30-49b1-a380-0b7971e75338)

### Code Reference

```
<div id="comment-[…]" class="chat-message " data-comment-id="[…]">
	<div class="chat-message-content">
		[…]
		<div class="chat-bubble">
			[…]
			<div class="chat-actions">
				<button class="chat-thread-toggle" data-comment-id="[…]" aria-expanded="true" aria-label="Collapse replies">[…]</button>

				<button class="chat-reply-button" data-comment-id="[…]" data-author="[…]" aria-label="[…]'">[…]</button>
			</div>
		</div>
	</div>

	<div class="chat-thread chat-thread-expanded" data-parent-id="[…]">[…]</div>
</div>
```

#### Further information:

1. [The "Intent" section of "Understanding Success Criterion 2.4.3: Focus Order | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html#intent)
